databases/qdrant/docker/embed-docs/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.11-slim-bookworm
```

Comment:
Same comment: Could you add to this file a step that simply builds the Dockerfile (just a `docker build`) so we have this added check?

https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/workflows/databases-qdrant-ci.yml

Reply:
databases/qdrant/docker/embed-docs
databases/qdrant/docker/chatbot
ci steps added

Reply:
Thank you!